api/blocklistener.go

```diff
@@ -8,26 +8,28 @@ import (
 	"github.com/iotexproject/iotex-proto/golang/iotexapi"
 	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 
+	apitypes "github.com/iotexproject/iotex-core/api/types"
 	"github.com/iotexproject/iotex-core/blockchain/block"
 	"github.com/iotexproject/iotex-core/pkg/log"
 )
 
-// blockListener defines the block listener in subscribed through API
-type blockListener struct {
-	stream  iotexapi.APIService_StreamBlocksServer
-	errChan chan error
+type StreamHandler func(interface{}) error
```

Comment:
this function is not helpful

Reply:
It acts as abbr for `func(interface{}) error`. Replace `StreamHandler` with `streamHandler` or `func(interface{}) error` ?